Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.8-alpine
+FROM python:3.8-buster
```

Comment:
The python version in the dockerfile is fine. The issue is with cloud sdk in the ci task.
https://github.com/ONSdigital/eq-survey-runner-benchmark/blob/master/ci/output-results-to-slack.yaml#L5-L6

Reply:
cloud sdk will only support up to 3.7 won't it?

Reply:
As per the cloud-sdk docs : `Cloud SDK requires Python. Supported versions are 3.5 to 3.7, and 2.7.9 or higher.`
`

Reply:
GCS supports 3.8. But I don't think we need the cloud sdk image for the ci task, just a python image.

Reply:
Yeah sorry that's what I was getting at - we shouldn't need it just for the Slack publish notification task